freqtrade/freqai/data_kitchen.py

```diff
@@ -1095,15 +1119,15 @@ def check_if_new_training_required(
 
         return retrain, trained_timerange, data_load_timerange
 
-    def set_new_model_names(self, pair: str, trained_timerange: TimeRange):
+    def set_new_model_names(self, pair: str, timestamp_id: int):
```

Comment:
@robcaulk , I've change this param because for training models, we are using the training end timestamp and for the new backtest_live_models features we need to use the start backtest timestamp. Using timestamp_id, we will handle this in a generic way.